### PR TITLE
fix typo error Preconditions.checkArgument in EncryptRule.getEncryptValues to be different with QueryAssistedEncryptAlgorithm

### PR DESCRIPTION
**Pull request by cheese8**
Changes proposed in this pull request:
- Preconditions.checkArgument message typo error in EncryptRule.getEncryptValues should be EncryptAlgorithm but QueryAssistedEncryptAlgorithm

### CODE REVIEW

**Comment by cheese8** [comment]
@strongduanmu little typo error fix

**Review by strongduanmu — CHANGES_REQUESTED** [review]
Hi @cheese8, can you fix ci error first?

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java [hunk 1]
```diff
@@ -231,7 +231,7 @@ public String getCipherColumn(final String logicTable, final String logicColumn)
     public List<Object> getEncryptAssistedQueryValues(final String schemaName, final String logicTable, final String logicColumn, final List<Object> originalValues) {
         Optional<EncryptAlgorithm> encryptor = findEncryptor(schemaName, logicTable, logicColumn);
         Preconditions.checkArgument(encryptor.isPresent() && encryptor.get() instanceof QueryAssistedEncryptAlgorithm,
-                String.format("Can not find QueryAssistedEncryptAlgorithm by %s.%s.", logicTable, logicColumn));
+                String.format("Cannot find QueryAssistedEncryptAlgorithm by %s.%s.", logicTable, logicColumn));
```
Comment (strongduanmu) [inline shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java:234]: @cheese8 Can we keep the original format?

**Review by strongduanmu — CHANGES_REQUESTED** [review]

File: shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java (hunk 1 above)
Comment (cheese8) [reply shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rule/EncryptRule.java:234]: just to test ci is ready or not

**Review by strongduanmu — APPROVED** [review]

**Pull request merged**